pvacseq/lib/parse_output.py

```diff
@@ -5,8 +5,18 @@
 import sys
 import os
 from math import ceil
+from statistics import median
 
-import pdb
+def prediction_method_lookup(prediction_method):
```

Comment:
Maybe this needs to be moved to a shared module of some sort as it appears here and in `main.py` as well as being used in the command line interface `choices`